cs.py

```diff
@@ -82,8 +86,15 @@ def main():
         from modules import azureaudit
         azureaudit.azure_audit()
         log.info("completed azure audit")
+        #exit(0)
+
+    
+    if args.number:
+        
+        from modules import manageReports
+        manageReports.numberOfReports(args.environment, args.number)
         exit(0)
 
 
```

Comment:
Extra lines not required. Can you please use `pylint` to check your code quality. I think we can make some improvements